actix-http/CHANGES.md

```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased] - xxx
 
+### Changed
+
+* Fix illegal chunked encoding. [#1611]
```

Comment:
this PR is #1615

Reply:
Thank you for the review.  bc62a4570f8472980481d5aa0e51b86fc74d3bac fixes the link.